cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -1667,39 +1667,38 @@ func (snr *sovereignNodeRunner) CreateManagedCryptoComponents(
 	return managedCryptoComponents, nil
 }
 
-func (snr *sovereignNodeRunner) CreateArgsRunTypeComponents() (*runType.ArgsSovereignRunTypeComponents, error) {
+// CreateArgsRunTypeComponents creates the arguments for sovereign runType components
+func (snr *sovereignNodeRunner) CreateArgsRunTypeComponents(coreComp mainFactory.CoreComponentsHandler) (*runType.ArgsSovereignRunTypeComponents, error) {
 	sovereignCfg := snr.configs.SovereignExtraConfig
 
 	codec := abi.NewDefaultCodec()
 	argsDataCodec := dataCodec.ArgsDataCodec{
 		Serializer: abi.NewSerializer(codec),
 	}
 
-	dataCodecHandler, err := dataCodec.NewDataCodec(argsDataCodec)
+	dataDecoderHandler, err := dataCodec.NewDataCodec(argsDataCodec)
```

Comment:
There are a lot of mixed namings for this codec.
Is it decoder or codec?

Reply:
`DataCodec` and `DataCodecHandler` everywhere should be now